Plugins/Flow.Launcher.Plugin.WebSearch/Main.cs

```diff
@@ -118,32 +117,31 @@ public async Task<List<Result>> QueryAsync(Query query, CancellationToken token)
         private async Task<IEnumerable<Result>> SuggestionsAsync(string keyword, string subtitle, SearchSource searchSource, CancellationToken token)
         {
             var source = _settings.SelectedSuggestion;
-            if (source != null)
+            if (source == null)
             {
-                //Suggestions appear below actual result, and appear above global action keyword match if non-global;
-                var score = searchSource.ActionKeyword == SearchSourceGlobalPluginWildCardSign ? scoreSuggestions : scoreSuggestions + 1;
+                return new List<Result>();
+            }
+            //Suggestions appear below actual result, and appear above global action keyword match if non-global;
+            var score = searchSource.ActionKeyword == SearchSourceGlobalPluginWildCardSign ? scoreSuggestions : scoreSuggestions + 1;
 
-                var suggestions = await source.Suggestions(keyword, token).ConfigureAwait(false);
+            var suggestions = await source.SuggestionsAsync(keyword, token).ConfigureAwait(false);
 
-                token.ThrowIfCancellationRequested();
+            token.ThrowIfCancellationRequested();
 
-                var resultsFromSuggestion = suggestions?.Select(o => new Result
+            var resultsFromSuggestion = suggestions?.Select(o => new Result
+            {
+                Title = o,
+                SubTitle = subtitle,
+                Score = score,
+                IcoPath = searchSource.IconPath,
+                Action = c =>
                 {
-                    Title = o,
-                    SubTitle = subtitle,
-                    Score = score,
-                    IcoPath = searchSource.IconPath,
-                    ActionKeywordAssigned = searchSource.ActionKeyword == SearchSourceGlobalPluginWildCardSign ? string.Empty : searchSource.ActionKeyword,
```

Comment:
why is this removed?

Reply:
It hasn't been removed, just move to a higher level so displayed like removal, 🤣

Reply:
no, this is actually removed, you can do a search, the original file has two of these, this pr file has only one.